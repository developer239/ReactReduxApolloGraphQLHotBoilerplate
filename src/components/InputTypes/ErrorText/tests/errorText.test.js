import React from 'react'
import { shallow } from 'enzyme'
import { theme } from 'styles'
import 'jest-styled-components'
import ErrorText from '../index'


describe('ErrorText Component', () => {
  it('has correct default styles', () => {
    const wrapper = shallow(
      <ErrorText theme={theme}>Some Content</ErrorText>,
    )
    expect(wrapper).toMatchSnapshot()
  })
})
