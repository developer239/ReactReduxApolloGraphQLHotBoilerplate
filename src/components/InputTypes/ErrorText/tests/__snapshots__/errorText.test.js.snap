// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorText Component has correct default styles 1`] = `
.c0 {
  display: inline-block;
  background-color: #ed5565;
  color: #fff;
  margin-top: 10px;
  padding: 10px;
  font-size: 12px;
}

<span
  className="c0"
>
  Some Content
</span>
`;
